input/vulnerabilities/QSEE_TrustZone.json

```diff
@@ -10,21 +10,24 @@
   "Reported_on" : [["2015-09-27", "qsee-blog"]],
   "Fixed_on" : [["2015-12-14", "qsee-blog"]],
   "Fix_released_on" : [["2015-12-14", "qsee-blog"]],
-  "Affected_versions" : [],
+  "Affected_versions" : [["5.0-5.1.1 and 6.0", "cvedetails-qsee"]],
   "Affected_devices" : [["Several, including Droid Turbo, Nexus 6 and Moto X 2nd Gen", "qsee-blog"]],
   "Affected_versions_regexp" : [],
```

Comment:
Need a regular expression for the versions so that we can test whether a device is affected automatically.

Reply:
Fixed